src/main/java/com/google/devtools/build/lib/runtime/BuildSummaryStatsModule.java

```diff
@@ -206,6 +237,25 @@ public void buildComplete(BuildCompleteEvent event) {
         criticalPathComputer = null;
       }
       profilePath = null;
+      cpuUserTimeForActions = 0;
+      cpuSystemTimeForActions = 0;
+      cpuTimeForBazelJvm = UNKNOWN;
+    }
+  }
+
+  private static String formatCpuTime(long milliseconds) {
+    if (milliseconds == UNKNOWN) {
+      return "???s";
+    } else {
+      return String.format("%.2fs", milliseconds / 1000.0);
+    }
+  }
+
+  private static long sumCpuTimes(long a, long b, long c) {
```

Comment:
Since you're hardcoding this to three parameters, you might as well name them with what kinds of CPU times they are.